lib/cfndsl/orchestration_template.rb

```diff
@@ -32,15 +32,15 @@ def create_types
 
             abreve_name = parts.join('_')
             if accessors.key? abreve_name
-              accessors.delete abreve_name # Delete potentially ambiguous names
+              accessors[abreve_name] = :duplicate # Delete potentially ambiguous names
```

Comment:
This never worked, If there were an even number of duplicates it would be removed, but an odd number would be included.  This is dangerous anyway as new types are being added all the time so something they was not a duplicate in a previous version will suddenly be removed when the specification is updated.